proceeding/text/idea.tex

```diff
@@ -18,3 +18,20 @@ \section{Idea}
 \item Open source, open data - allows for 1. users to download any data from our website, and 2. for other developers to contribute to the code.
 
 \end{itemize}
+
+
+\paragraph{}    gamma-sky.net is a one-stop resource for perusing images and catalogs but also closely examining a specific gamma-ray source.
+    Although it was mainly built for the astronomical community, the webpage also targets the general public through a
+    user-friendly interface and a clean information layout, all of which is compiled under cutting-edge web tools. \par
+
+\paragraph{}    Individuals who access the website via any modern internet browser will be welcomed with the Map View page.
+    We overlaid multi-wavelength survey images, most of which are all-sky images, onto a three-dimensional sphere.
+    The map features pan-and-zoom functionality for easily navigating and quickly browsing the sky. Gamma-ray sources from
+    our catalog data have been pinpointed onto the sphere, as shown in Figure 1. This setup allows the user to understand
```

Comment:
You should use `\ref` to refer to the Figure instead of hard-coding the number 1.
https://en.wikibooks.org/wiki/LaTeX/Labels_and_Cross-referencing